app/page/home/index.tsx

```diff
@@ -19,12 +28,35 @@ export enum InfoNavigation {
 
 export function Home() {
   const hasNoAccount = useAccounts().length === 0
+
+  const { provider } = useContext(ProviderContext)
+  const { createSimpleAccount } = useAction()
+  const network = useNetwork()
+
+  useEffect(() => {
+    const initSimpleAccount = async () => {
+      const hasSimpleAccountFactory = address.isValid(
+        network.accountFactory[AccountType.SimpleAccount]
+      )
+
+      if (hasNoAccount && hasSimpleAccountFactory) {
+        const account = await SimpleAccount.initWithFactory(provider, {
+          ownerPrivateKey: Wallet.createRandom().privateKey,
+          salt: number.random(),
+          factoryAddress: network.accountFactory[AccountType.SimpleAccount]
+        })
+
+        await createSimpleAccount(`Account 1`, account, network.id)
+      }
+    }
+
+    initSimpleAccount()
+  }, [network.id])
+
   return (
     <>
       <Navbar />
-      {hasNoAccount ? (
-        <AccountCreation />
-      ) : (
+      {!hasNoAccount && (
```

Comment:
nit: I think navbar can be shown only when at least one account exists, so we don't need to handle no account case in `AccountSelector`

Reply:
We’re not considering the case with a PasskeyAccountFactory but without a SimpleAccountFactory, right?